api/backend.go

```diff
@@ -415,6 +421,12 @@ func (b *StatusBackend) SelectAccount(address, password string) error {
 	if err != nil {
 		return err
 	}
+	// Re-register the SendTransaction RPC handler so that it uses the
```

Comment:
The current solution to passing the pwd to the sendtransaction method RPC handler is to add the password as an argument to the b.registerHandlers() method and calling it again when an account is selected.